docs/src/app/components/PropTypeDescription.jsx

```diff
@@ -112,11 +112,28 @@ const PropTypeDescription = React.createClass({
         defaultValue = prop.defaultValue.value.replace(/\n/g, '');
       }
 
+      if (prop.required) {
+        key = `<span class="requiredProp">${key} \*</span>`;
```

Comment:
Erm, why do we still need a `className` with an inline style? :beer: :beer: :beers: :beers: :notes: :dancers: :facepunch: :oncoming_police_car: :cop: :innocent: :zzz: